src/unbounded.rs

```diff
@@ -149,6 +149,22 @@ pub struct Unbounded<T> {
 
 impl<T> Unbounded<T> {
     /// Creates a new unbounded queue.
+    #[cfg(not(loom))]
```

Comment:
this sort of code duplication is very unfortunate, could we encapsulate that into a macro? (just the cfg + const switch)

Reply:
(Feel free to copy [a helper macro used in my crates](https://github.com/taiki-e/atomic-maybe-uninit/commit/557111d24bbec3d84d1dcf9559cbb110a0312450#diff-23af5356f6001cd3993cd3c801fb7716ea02d1e504081b9fb569332db6107e80R40-R54) if needed)

Reply:
Done.